### DISCUSSION TITLE
Who ✍️⬜🐳?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#997] [Galaxy Trivia] [answered]
**Bonus:** What Fortune 500 company was named after one of its main characters?

**Comment by peterbaccaro** [comment, marked as answer]
There are references to a white whale in **Herman Melville**'s classic novel "Moby-Dick." In Melville's book, the white whale is named Moby Dick and serves as a symbol of obsession and the destructive pursuit of vengeance. Herman Melville is the author of "Moby-Dick," which was first published in 1851.

The Fortune 500 company that was named after a character from Herman Melville's novel "Moby-Dick" is **Starbucks**. The company's name was inspired by the first mate on the Pequod, Starbuck. The founders of Starbucks, Gordon Bowker, Zev Siegl, and Jerry Baldwin, were fans of the novel and chose the name for their coffee company, which was first established in Seattle in 1971.
